tests/integration-tests/tests/storage/test_efs/test_multiple_efs/pcluster.config.yaml

```diff
@@ -38,7 +38,8 @@ Scheduling:
           MaxCount: 1
       Networking:
         SubnetIds:
-          - {{ private_subnet_ids[2] }}
+          # us-isob-east-1 provides 2 availability zones.
```

Comment:
NIT: Is it possible to apply this logic based on the length of `private_subnet_ids` instead of a specific region?

Reply:
I agree: will do in next rev